framework/src/interfaces/TaggingInterface.C

```diff
@@ -379,8 +408,12 @@ TaggingInterface::assignTaggedLocalResidual()
 void
 TaggingInterface::accumulateTaggedLocalMatrix()
 {
-  for (auto & ke : _ke_blocks)
-    *ke += _local_ke;
+  if (_eigen_problem && _eigen_problem->negativeSignEigenKernel())
```

Comment:
Seems like it should be `!_eigen_problem->negativeSignEigenKernel()`? If the negative sign is already encoded in the kernel residual/Jacobian, then in the tagging interface we should do `+=`. E.g. this should be consistent with the sign that you're giving in the residual methods, should it not? If not, then we will need a comment about why the behavior is different

Reply:
Moreover, I am still concerned that these factors should not be applied if this is an interface for an Ax kernel

Reply:
This is to remove a matrix scaling with factor -1 later.